cl/opinion_page/views.py

```diff
@@ -723,6 +743,7 @@ async def view_recap_document(
             "timezone": COURT_TIMEZONES.get(d.court_id, "US/Eastern"),
             "redirect_to_pacer_modal": redirect_to_pacer_modal,
             "authorities": await rd.cited_opinions.aexists(),
+            "prayer_eligible": prayer_is_eligible,
```

Comment:
It seems this context variable is not being used in any template. It's also present in `view_docket`.

Reply:
You're correct! Thanks for bringing this to my attention.